css_final.css

```diff
@@ -1,112 +1,96 @@
 * {
-    margin: 0px;
-    padding: 0;
-    font-family: Arial, Helvetica, sans-serif;
-
+  margin: 0px;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
 }
 
-
 .heading {
-    display: flex;
-    background-color: #232f3e;
-    box-shadow: 0px 1px 2px #232f3e;
-
+  display: flex;
+  background-color: #232f3e;
+  box-shadow: 0px 1px 2px #232f3e;
 }
 h1 {
-    color: coral;
-    font-weight: bold;
-    
-    background: transparent;
-    padding: 7px;
-    
+  color: coral;
+  font-weight: bold;
+
+  background: transparent;
+  padding: 7px;
 }
 
 body {
-    display: flex;
-    justify-content: center;
+  display: flex;
+  justify-content: center;
 }
 
 .outer-wrapper {
-    margin-top: 40px;
-    margin-left: 20px;
-    margin-right: 20px;
-    border: 1px solid black;
-    border-radius: 4px;
-    box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.9);
-    max-width: 90%;
-    max-height: fit-content;
-
-    
+  margin-top: 40px;
+  margin-left: 20px;
+  margin-right: 20px;
+  padding-bottom: 5%;
```

Comment:
your code is good to go Anjali @anjalisingh23 just remove this padding here

Reply:
I added padding cuz adding new row was causing overflow
Shall I still remove it?

Reply:
yes ! I love your PR but if you are adding only a opportunity you should just only edit the index.html file Anjali! 

i would edit only the index.html file and submit a new PR 

Tag me if you need any assitance !

Reply:
@souvikrajsingh  I commited css file but how to submit a new PR?

Reply:
do it from the beginning anjali, delete the local copy you have of this repo on your system and fork and clone it again.

Work ONLY on the index.html file.

Just add the name of the opportunity as directed and you are good to go! 
Cheers

Reply:
okh got it
Thankyou